From 9cbc91ca6d5ded47e65fe424c39cbe43511a1bd0 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Fri, 25 Jul 2025 11:02:33 +0200
Subject: [PATCH] mailbox: fix misspellings of "STMicroelectronics"

"STMicroelectronics" company name is misspelled in the description of
STI_MBOX and STM32_IPCC configurations. Fix these misspellings.

Change-Id: Ibe2e5082da4b89b7b178adecf596cf68c97681b4
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/477731
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 drivers/mailbox/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/mailbox/Kconfig
+++ b/drivers/mailbox/Kconfig
@@ -130,7 +130,7 @@ config STI_MBOX
 	tristate "STI Mailbox framework support"
 	depends on ARCH_STI && OF
 	help
-	  Mailbox implementation for STMicroelectonics family chips with
+	  Mailbox implementation for STMicroelectronics family chips with
 	  hardware for interprocessor communication.
 
 config TI_MESSAGE_MANAGER
@@ -235,7 +235,7 @@ config STM32_IPCC
 	tristate "STM32 IPCC Mailbox"
 	depends on ARCH_STM32 || COMPILE_TEST
 	help
-	  Mailbox implementation for STMicroelectonics STM32 family chips
+	  Mailbox implementation for STMicroelectronics STM32 family chips
 	  with hardware for Inter-Processor Communication Controller (IPCC)
 	  between processors. Say Y here if you want to have this support.
 
